features/FEATURE_LWIP/lwip-interface/lwip_stack.c

```diff
@@ -329,7 +329,7 @@ const char *lwip_get_netmask(char *buf, int buflen)
 #if LWIP_IPV4
     const ip4_addr_t *addr = netif_ip4_netmask(&lwip_netif);
     if (!ip4_addr_isany(addr)) {
-        return inet_ntoa_r(addr, buf, buflen);
+        return ip4addr_ntoa_r(ip_2_ip4(addr), buf, buflen);
```

Comment:
This looks wrong. addr is ip4_addr_t, so ip_2_ip4 shouldn't be used on it.

Tested it with IPv6 enabled?

Is inet_ntoa_r colliding with something?

    return ip4addr_ntoa_r(addr, buf, buflen);

should be correct, I think.